ibm_catalog.json

```diff
@@ -182,6 +222,36 @@
                   }
                 }
               },
+              {
+                "key": "cspm_enabled",
```

Comment:
`cspm_enabled` is a boolean - you don't need t add a manual dropdown for booleans